src/mumble/Settings.cpp

```diff
@@ -461,6 +462,7 @@ BOOST_TYPEOF_REGISTER_TYPE(Settings::OverlayShow)
 BOOST_TYPEOF_REGISTER_TYPE(Settings::ProxyType)
 BOOST_TYPEOF_REGISTER_TYPE(Settings::ChannelExpand)
 BOOST_TYPEOF_REGISTER_TYPE(Settings::ChannelDrag)
+BOOST_TYPEOF_REGISTER_TYPE(Settings::UserDrag)
```

Comment:
Forgot to remove this line when I removed the introduction of UserDrag, removed and squashed to the second commit.